packages/yoastseo/spec/scoring/assessments/inclusiveLanguage/configuration/sesAssessmentsSpec.js

```diff
@@ -20,7 +20,7 @@ describe( "SES assessments", function() {
 			expect( assessmentResult.getScore() ).toEqual( 3 );
 			expect( assessmentResult.getText() ).toEqual(
 				"Avoid using <i>illegal immigrants</i> as it is potentially harmful. " +
-				"Consider using an alternative, such as <i>undocumented people</i>. " +
+				"Consider using an alternative, such as <i>undocumented people, people without papers, immigrants without papers</i>. " +
```

Comment:
This change has been covered in this PR: https://github.com/Yoast/wordpress-seo/pull/19312

Reply:
but I think it's not a harmful to leave it as is here